src/Task/Base/Exec.php

```diff
@@ -67,11 +67,22 @@ class Exec extends BaseTask implements CommandInterface, PrintedInterface, Simul
     protected $process;
 
     /**
+     * @var resource|string
+     */
+    protected $input;
+
+    /**
+     * @var boolean
+     */
+    protected $interactive = false;
+
+    /**
      * @param string|\Robo\Contract\CommandInterface $command
      */
     public function __construct($command)
     {
         $this->command = $this->receiveCommand($command);
+        $this->interactive = posix_isatty(STDOUT);
```

Comment:
posix_isatty might not be available; this function should be guarded by a `function_exists()`